app/about_us/OurPartners.js

```diff
@@ -0,0 +1,24 @@
+const OurPartners = () => {
```

Comment:
Attention!
You are not exporting anything here, when you'll try to import it the app will crash